# HG changeset patch
# User Vadim Yanitskiy <fixeria@osmocom.org>
# Date 1716481958 -25200
#      Thu May 23 23:32:38 2024 +0700
# Node ID 1882f20bad701f1315834104e8d89e48da8e075b
# Parent  1d144553a5d0c6c3e1d255bd4d8019d78cb9ce0d
rvinterf: fix -Werror=format-security in get_pirelli_imei()

diff -r 1d144553a5d0 -r 1882f20bad70 rvinterf/etmsync/pirimei.c
--- a/rvinterf/etmsync/pirimei.c	Tue Dec 12 07:45:56 2023 +0000
+++ b/rvinterf/etmsync/pirimei.c	Thu May 23 23:32:38 2024 +0700
@@ -30,7 +30,7 @@
 	cl_des(ciphertext[0], dieid_key, decrypted[0], CL_DES_DECRYPTION);
 	cl_des(ciphertext[1], dieid_key, decrypted[1], CL_DES_DECRYPTION);
 	if (bcmp(decrypted[1], dieid_key, 8)) {
-		printf(failmsg);
+		puts(failmsg);
 		return(ERROR_TARGET);
 	}
 	bcopy(decrypted[0], pirelli_imeisv, 8);
